src/include/connection.h

```diff
@@ -811,38 +811,42 @@ struct __wt_connection_impl {
  */
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_TIMING_STRESS_AGGRESSIVE_STASH_FREE 0x00000001ull
-#define WT_TIMING_STRESS_AGGRESSIVE_SWEEP 0x00000002ull
-#define WT_TIMING_STRESS_BACKUP_RENAME 0x00000004ull
-#define WT_TIMING_STRESS_CHECKPOINT_EVICT_PAGE 0x00000008ull
-#define WT_TIMING_STRESS_CHECKPOINT_HANDLE 0x00000010ull
-#define WT_TIMING_STRESS_CHECKPOINT_SLOW 0x00000020ull
-#define WT_TIMING_STRESS_CHECKPOINT_STOP 0x00000040ull
-#define WT_TIMING_STRESS_COMMIT_TRANSACTION_SLOW 0x00000080ull
-#define WT_TIMING_STRESS_COMPACT_SLOW 0x00000100ull
-#define WT_TIMING_STRESS_EVICT_REPOSITION 0x00000200ull
-#define WT_TIMING_STRESS_FAILPOINT_EVICTION_SPLIT 0x00000400ull
-#define WT_TIMING_STRESS_FAILPOINT_HISTORY_STORE_DELETE_KEY_FROM_TS 0x00000800ull
-#define WT_TIMING_STRESS_HS_CHECKPOINT_DELAY 0x00001000ull
-#define WT_TIMING_STRESS_HS_SEARCH 0x00002000ull
-#define WT_TIMING_STRESS_HS_SWEEP 0x00004000ull
-#define WT_TIMING_STRESS_PREFETCH_1 0x00008000ull
-#define WT_TIMING_STRESS_PREFETCH_2 0x00010000ull
-#define WT_TIMING_STRESS_PREFETCH_3 0x00020000ull
-#define WT_TIMING_STRESS_PREFIX_COMPARE 0x00040000ull
-#define WT_TIMING_STRESS_PREPARE_CHECKPOINT_DELAY 0x00080000ull
-#define WT_TIMING_STRESS_PREPARE_RESOLUTION_1 0x00100000ull
-#define WT_TIMING_STRESS_PREPARE_RESOLUTION_2 0x00200000ull
-#define WT_TIMING_STRESS_SLEEP_BEFORE_READ_OVERFLOW_ONPAGE 0x00400000ull
-#define WT_TIMING_STRESS_SPLIT_1 0x00800000ull
-#define WT_TIMING_STRESS_SPLIT_2 0x01000000ull
-#define WT_TIMING_STRESS_SPLIT_3 0x02000000ull
-#define WT_TIMING_STRESS_SPLIT_4 0x04000000ull
-#define WT_TIMING_STRESS_SPLIT_5 0x08000000ull
-#define WT_TIMING_STRESS_SPLIT_6 0x10000000ull
-#define WT_TIMING_STRESS_SPLIT_7 0x20000000ull
-#define WT_TIMING_STRESS_SPLIT_8 0x40000000ull
-#define WT_TIMING_STRESS_TIERED_FLUSH_FINISH 0x80000000ull
+#define WT_TIMING_STRESS_AGGRESSIVE_STASH_FREE 0x000000001ull
+#define WT_TIMING_STRESS_AGGRESSIVE_SWEEP 0x000000002ull
+#define WT_TIMING_STRESS_BACKUP_RENAME 0x000000004ull
+#define WT_TIMING_STRESS_CHECKPOINT_EVICT_PAGE 0x000000008ull
+#define WT_TIMING_STRESS_CHECKPOINT_HANDLE 0x000000010ull
+#define WT_TIMING_STRESS_CHECKPOINT_SLOW 0x000000020ull
+#define WT_TIMING_STRESS_CHECKPOINT_STOP 0x000000040ull
+#define WT_TIMING_STRESS_CLOSE_STRESS_LOG 0x000000080ull
+#define WT_TIMING_STRESS_COMMIT_TRANSACTION_SLOW 0x000000100ull
+#define WT_TIMING_STRESS_COMPACT_SLOW 0x000000200ull
+#define WT_TIMING_STRESS_EVICT_REPOSITION 0x000000400ull
+#define WT_TIMING_STRESS_FAILPOINT_EVICTION_SPLIT 0x000000800ull
+#define WT_TIMING_STRESS_FAILPOINT_HISTORY_STORE_DELETE_KEY_FROM_TS 0x000001000ull
+#define WT_TIMING_STRESS_HS_CHECKPOINT_DELAY 0x000002000ull
+#define WT_TIMING_STRESS_HS_SEARCH 0x000004000ull
+#define WT_TIMING_STRESS_HS_SWEEP 0x000008000ull
+#define WT_TIMING_STRESS_LIVE_RESTORE_CLEAN_UP 0x000010000ull
+#define WT_TIMING_STRESS_OPEN_INDEX_SLOW 0x000020000ull
```

Comment:
This and `WT_TIMING_STRESS_LIVE_RESTORE_CLEAN_UP` isn't used in the new change I believe, can also be removed